python-frontend/src/main/java/org/sonar/python/semantic/SymbolTableBuilder.java

```diff
@@ -173,6 +174,26 @@ private void addSymbolsToTree(FileInputImpl fileInput) {
     }
   }
 
+  private static void handleClassMemberAccess(FileInput fileInput) {
+    BaseTreeVisitor classMemberAccess = new BaseTreeVisitor() {
+      @Override
+      public void visitQualifiedExpression(QualifiedExpression qualifiedExpression) {
+        super.visitQualifiedExpression(qualifiedExpression);
+        if (qualifiedExpression.qualifier() instanceof HasSymbol) {
+          Symbol symbol = ((HasSymbol) qualifiedExpression.qualifier()).symbol();
+          if (symbol != null && symbol.kind() == Symbol.Kind.CLASS) {
+            Symbol member = ((ClassSymbolImpl) symbol).membersByName().get(qualifiedExpression.name().name());
```

Comment:
I don't think that's correct. I believe you should use `resolveMember` rather than exposing `membersByName`.
```
class A:
  def f(): print("A.f")

class B(A):
  pass

B.f()
```

Reply:
Good point. My wrong assumption was that static members were not inherited